docs/docs/dev-server/node-api.md

```diff
@@ -73,6 +73,46 @@ async function main() {
 main();
 ```
 
+## Combine with your own CLI definitions
+
+If you extend the dev-server and want to use `command-line-args` to add your own CLI definitions, it is recommended to use the [`partial` option](https://github.com/75lb/command-line-args/wiki/Partial-parsing).
+
+```js
+const myDefinitions = [
+  {
+    name: 'foo',
+    type: String,
+    description:
+      'Bar',
+  },
+];
+
+const myConfig = ...commandLineArgs(myServerDefinitions, { partial: true });
```

Comment:
why the `...` here?

Reply:
Erhh oops that's a copy paste error, since in my config in my code I merge it with defaults ;). Changed it